docs/src/main/paradox/play-framework.md

```diff
@@ -28,15 +28,21 @@ With the following `helloworld.proto` file:
 
 @@snip[helloworld.proto]($root$/../play-interop-test-scala/src/main/protobuf/helloworld.proto) { #protoSources }
 
-The module file is generated in `example.myapp.helloworld.grpc.helloworld.AkkaGrpcClientModule`.
+The module file is generated in @scala[`example.myapp.helloworld.grpc.helloworld.AkkaGrpcClientModule` by default for Scala]
+@java[`example.myapp.helloworld.grpc.AkkaGrpcClientModule` by default for Java], which corresponds to the default value
```

Comment:
the package is different since the flat_package setting is different for the langs

Reply:
I think we may have messed it up... digging... The package was definitely different and not the same on default settings and docs which i was following

Reply:
for `play-interop-test-scala` the package is different because there we don't use the plugin, but we use `ReflectiveCodeGen`, which sets `flat_package` back to `false`.

Perhaps we should avoid being so subtle and just split up our interop-test subproject into separate scala/java/common projects... more subprojects, but less complicated ones.

Reply:
Would be nice I agree (the splitting up into packages)